### DISCUSSION TITLE
Welcome to GoutPal Discussions

### DISCUSSION THREAD

**Discussion by kct2020** [kct2020/goutpal-com-skeleventy#25] [Announcements]
This replaces my [Current GoutPal Forum](https://goutpal.net/forums/). So please use it to share your questions, experiences, and opinions about any aspect of gout.

However, if you want to discuss a specific GoutPal page, please use the [Issues tab above](https://github.com/kct2020/goutpal-com-skeleventy/issues). Also use Issues if you need clarification about using any GoutPal services.

**Comment by kct2020** [comment]
Today, I'd like to explain why I'm moving the forum to GitHub. 

Simply - because it's too much work. Especially using a self-hosted WordPress forum. Which I've used in various forms since my first GoutPal forum. Over the years, there have been lots of insightful discussions. And I'm blessed with hundreds of personal thank you messages.

Most importantly for me, my discussions with like-minded people have allowed me to keep GoutPal content relevant to current gout sufferer concerns. So your feedback remains vital to me. But as I said, the WordPress forum just takes too much administration time. Which severely impacts the time I have for content development.

So the advantages of GitHub are...
- Removes the cost of hosting.
- Is a safe, secure environment.
- Has outstanding technical support for users.
- Allows effective collaboration. Both in resolving issues and in administering the platform.

I worry that GitHub's roots in technical development might distract some users. But I hope we can overcome any specific concerns as they arise. In particular, if you want a simpler view of contributions, I highly recommend installing the [GitHub Mobile App](https://github.com/mobile).

## GoutPal Contributors ##
A long time ago, I started opining that I ran GoutPal for visitors. So this move takes your involvement to the next level. But only as far as you want to go. Because your involvement levels are:

### Anonymous Reader ###
As always, you can get value from reading these GoutPal Discussions, the associated GoutPal Issues, or my [gout information website](https://goutpal.com/). In return, I see which topics attract most interest. So that helps me focus my attention on the gout concerns that matter to you. For example, I can see that most visitors to this repository are interested in GoutPal Discussions. Despite there being only this single topic. So that prompted me to explain more about what I'm doing here.

### GoutPal Contributor ###
Though I may develop different contributor levels in the future, you are currently a GoutPal Contributor or not. So the ways you can contribute are:

- Add comments to these GoutPal Discussions. This can include all of your questions, experiences, or opinions about the gout topic being discussed. But remember to be a good contributor by staying on-topic. Because if you want to chat about a different aspect of your gout...
- Start a [New GoutPal Discussion](https://github.com/kct2020/goutpal-com-skeleventy/discussions/new). Which can be all your questions, experiences, or opinions about any gout topic.
- Add comments to any [GoutPal Issue](https://github.com/kct2020/goutpal-com-skeleventy/issues). As with discussions, you should keep your comments on-topic. Remembering that the topic is about a specific GoutPal page.
- Create a [New GoutPal Issue](https://github.com/kct2020/goutpal-com-skeleventy/issues/new/choose). This has to be about the content of a given page. Unless you have an idea for a new page.

## It's Your GoutPal ##
Please remember that by contributing to GoutPal Issues, you are making a very high level of contribution to help other gout sufferers. Because many more sufferers share your gout concerns. But it takes a special kind of person to speak out on behalf of others.

**Comment by hfat** [comment]
Hi Keith! I'm nobody on your forum, and just wanted to let you know I'm fine using Github.
Thanks for all the work you put in over the years!

**Reply by kct2020** [reply]
You've made my day, my week, my month, and even my life (or something like that).

So good that you're here. But it's the end of an era for "nobody knows" jokes! And I can't immediately think of a Horselover Fat joke - but I like a challenge.

It's me who must thank you. You are far more patient with forum replies than I can ever be. And extremely responsive. I hope I can try harder to be more communicative this year.

**Comment by metamorph45** [comment]
Hi Keith, 
It's me, "metamorph" here again, after a very long absence from Goutpal Discussions.  I am now back as "metamorph45" because I am not too sure whether my old login records are still valid.
Over the many years of absence from GoutPal, I had gone through lots of ups and downs. My worst health problem was having to go through a heart bypass (CABD5) in 2012. I also had intermittent bouts of gout flareups due to my love of rich protein food which I could resist. But I am not too concerned or worried about them because of my unwavering faith in BBB. I am still up and around.
Just a few days ago I had gout flareups on both feet (due to too much of purine-rich food – mushrooms, mussels. etc.). The pain and flareup gradually diminished and were gone the next day.
I am wondering how "Black Bean Broth" is doing for other gout sufferers. To this very day, my faith in BBB is still very strong. 
Now we have to contend with the highly infectious and fast spreading Covid-19 and it's variants.

**Reply by kct2020** [reply]
Hey metamorph, really great to hear from you.

Sorry to learn that you needed a bypass. But my experience from other friends is that it gave them a new lease of life. So I hope that applies to you.

Black Bean Broth information remains very popular with GoutPal visitors. Unfortunately, discussions have dwindled over the years. But I'm trying new ways to encourage comments. So hopefully, things will slowly improve.
